tier1/{{cookiecutter.project_slug}}/repolinter.json

```diff
@@ -0,0 +1,763 @@
+          "options": {
+            "globsAll": [
+              "README.md"
+            ],
+            "content": "Coding Style and Linters",
+            "flags": "i"
+          }
+        }
+      },
+      "readme-contains-branching-model": {
+        "level": "warning",
+        "rule": {
+          "type": "file-contents",
+          "options": {
+            "globsAll": [
+              "README.md"
+            ],
+            "content": "Branching Model",
+            "flags": "i"
+          }
+        }
+      },
+      "readme-contains-contributing": {
+        "level": "warning",
+        "rule": {
+          "type": "file-contents",
+          "options": {
+            "globsAll": [
+              "README.md"
+            ],
+            "content": "Contributing",
+            "flags": "i"
+          }
+        }
+      },
+      "readme-contains-codeowners": {
+        "level": "warning",
+        "rule": {
+          "type": "file-contents",
+          "options": {
+            "globsAll": [
+              "README.md"
+            ],
+            "content": "Codeowners",
+            "flags": "i"
+          }
+        }
+      },
+      "readme-contains-community": {
+        "level": "warning",
+        "rule": {
+          "type": "file-contents",
+          "options": {
+            "globsAll": [
+              "README.md"
+            ],
+            "content": "Community",
+            "flags": "i"
+          }
+        }
+      },
+      "readme-contains-community-guidelines": {
+        "level": "warning",
+        "rule": {
+          "type": "file-contents",
+          "options": {
+            "globsAll": [
+              "README.md"
+            ],
+            "content": "Community Guidelines",
+            "flags": "i"
+          }
+        }
+      },
+      "readme-contains-governance": {
+        "level": "off",
+        "rule": {
+          "type": "file-contents",
+          "options": {
+            "globsAll": [
+              "README.md"
+            ],
+            "content": "Governance",
+            "flags": "i"
+          }
+        }
+      },
+      "readme-contains-feedback": {
+        "level": "warning",
+        "rule": {
+          "type": "file-contents",
+          "options": {
+            "globsAll": [
+              "README.md"
+            ],
+            "content": "Feedback",
+            "flags": "i"
+          }
+        }
+      },
+      "readme-contains-glossary": {
+        "level": "warning",
+        "rule": {
+          "type": "file-contents",
+          "options": {
+            "globsAll": [
+              "README.md"
+            ],
+            "content": "Glossary",
+            "flags": "i"
+          }
+        }
+      },
+      "readme-contains-policies": {
+        "level": "error",
+        "rule": {
+          "type": "file-contents",
+          "options": {
+            "globsAll": [
+              "README.md"
+            ],
+            "content": "Policies",
```

Comment:
@decause-gov Should we also include checks for the Policies subsections (Open Source Policy, Security and Responsible Disclosure Policy)? It's important content that we should make sure doesn't get left out just in case

Reply:
Yes, I agree. Having those sections in the READMEs is the default level for Tier1, and even if Tier2+ has that info in separate files, it is a good practice to reference the individual files from within the README. Good catch Nat.

Reply:
Added individual rules at tier1 for both the policies